### PR TITLE
feat: mandate changes for Saved Card flow

### PR DESCRIPTION
**Pull request by PritishBudhiraja**

### CODE REVIEW

File: src/Components/SaveDetailsCheckbox.res [hunk 1]
```diff
@@ -46,13 +48,21 @@ let make = (~isChecked, ~setIsChecked) => {
     ? ("Checkbox--checked", "CheckboxInput--checked", "CheckboxLabel--checked")
     : ("", "", "")
 
+  let valueToBeRendered = if showFields {
+    localeString.saveCardDetails
+  } else if business.name !== "" {
```
Comment (vsrivatsa-juspay) [inline src/Components/SaveDetailsCheckbox.res:53]: use this conditional logic while setting business.name in option atom

**Review by vsrivatsa-juspay — CHANGES_REQUESTED** [review]
Requested a change

File: src/Components/SaveDetailsCheckbox.res (hunk 1 above)
Comment (PritishBudhiraja) [reply src/Components/SaveDetailsCheckbox.res:53]: Updated

File: src/Components/SaveDetailsCheckbox.res [hunk 2]
```diff
@@ -46,13 +48,19 @@ let make = (~isChecked, ~setIsChecked) => {
     ? ("Checkbox--checked", "CheckboxInput--checked", "CheckboxLabel--checked")
     : ("", "", "")
 
+  let valueToBeRendered = if showFields {
```
Comment (vsrivatsa-juspay) [inline src/Components/SaveDetailsCheckbox.res:51]:
```suggestion
  let saveCardCheckboxLabel = if showFields {
```

File: src/Components/SaveDetailsCheckbox.res [hunk 3]
```diff
@@ -46,13 +48,19 @@ let make = (~isChecked, ~setIsChecked) => {
...
   <div className={`Checkbox ${checkboxState} flex flex-row gap-2 items-center`}>
     <style> {React.string(css)} </style>
     <label className={`container CheckboxInput ${checkedState}`}>
       <input type_={`checkbox`} onChange />
-      <div className={`checkmark CheckboxInput ${checkedState}`} />
-      <div className={`CheckboxLabel ${checkBoxLabelSate} ml-2`}>
-        {React.string(localeString.saveCardDetails)}
+      <div className={`checkmark CheckboxInput ${checkedState} mt-1`} />
+      <div className={`CheckboxLabel ${checkBoxLabelSate} ml-2 w-11/12`}>
+        {React.string(valueToBeRendered)}
```
Comment (vsrivatsa-juspay) [inline src/Components/SaveDetailsCheckbox.res:63]:
```suggestion
        {React.string(saveCardCheckboxLabel)}
```

**Review by vsrivatsa-juspay — APPROVED** [review]

**Review by vsrivatsa-juspay — APPROVED** [review]

**Review by prafulkoppalkar — APPROVED** [review]

**Review by ArushKapoorJuspay — APPROVED** [review]

**Pull request merged**